Makefile

```diff
@@ -15,16 +15,15 @@ src/commcare_cloud/help_cache/ansible-playbook.txt: export ANSIBLE_CONFIG=src/co
 src/commcare_cloud/help_cache/ansible-playbook.txt:
 	$(ANSIBLE_ENV) ansible-playbook -h > src/commcare_cloud/help_cache/ansible-playbook.txt
 
-docs/commcare-cloud/commands/index.md : src/commcare_cloud/* src/commcare_cloud/*/* src/commcare_cloud/*/*/*
-	manage-commcare-cloud make-docs > docs/commcare-cloud/commands/index.md
+hosting_docs/source/reference/1-commcare-cloud/commands.md : src/commcare_cloud/* src/commcare_cloud/*/* src/commcare_cloud/*/*/*
+	manage-commcare-cloud make-docs > hosting_docs/source/reference/1-commcare-cloud/commands.md
 
 
-
-docs/changelog/%.md : changelog/%.yml src/commcare_cloud/manage_commcare_cloud/*
+hosting_docs/source/changelog/%.md : changelog/%.yml src/commcare_cloud/manage_commcare_cloud/*
 	manage-commcare-cloud make-changelog $< > $@
 
-docs/changelog/index.md : changelog/*.yml src/commcare_cloud/manage_commcare_cloud/*
-	manage-commcare-cloud make-changelog-index > docs/changelog/index.md
+hosting_docs/source/changelog/index.md : changelog/*.yml src/commcare_cloud/manage_commcare_cloud/*
+	manage-commcare-cloud make-changelog-index > hosting_docs/source/changelog/index.md
```

Comment:
For these three changes, It looks like `make` is set up to now only write to the new location (`hosting_docs/source/`).

I think the PR in which we stop writing to the old location (`docs/`) from `make` should also be the PR in which we delete the docs in the old location.

My preference would be to have it write to both in this PR, and then later when we sunset the old docs location we can make `make` not write to the old location and in that same PR we can delete the files in the old location.

If you agree with that, then my suggestion here in this PR would be to keep the lines that were there previously and simply _add_ these new entries as well, making `make` write to both the old and the new locations.

Reply:
Done

Reply:
Awesome, I tested this out locally and it seems to work as desired! Unrelated to your changes, while I was looking at it, I found a stray file in docs/changelog we can remove, which I did in https://github.com/dimagi/commcare-cloud/pull/5313